### ISSUE TITLE
Nanoplot results

### ISSUE THREAD

**Issue by QuentinPerriere** [wdecoster/NanoPlot#393]
Thank you again for this tool,

I tested again NanoPlot on my data, and it showed that only 657 reads had a Phred score greater than Q10 ( I initially have 51653 reads in this fastq.gz file)
However, when I ran Fastp with the command fastp -i input_path -o fastp_output_path -A -G --qualified_quality_phred 10 --thread 8 followed by Seqtk using seqtk sample -s100 fastp_output_path 7000 | gzip > seqtk_output_path, I successfully extracted and generated a file containing 7000 reads with Q >10.

I use the latest version 1.42.0 of nanoplot

**Comment by wdecoster** [comment]
Yes, there are two ways to average quality scores across a read ([see my blog post](https://gigabaseorgigabyte.wordpress.com/2017/06/26/averaging-basecall-quality-scores-the-right-way/)), and I would argue that NanoPlot does this correctly.

**Comment by QuentinPerriere** [comment]
> Yes, there are two ways to average quality scores across a read ([see my blog post](https://gigabaseorgigabyte.wordpress.com/2017/06/26/averaging-basecall-quality-scores-the-right-way/)), and I would argue that NanoPlot does this correctly.

thank you for responding. but what I should do in this case ? fastp and Seqtk results are not reflected in the QC report generated by NanoPlot. This inconsistency is causing confusion in my upcoming presentation, as the QC metrics do not align with the actual quality of my filtered data.

**Comment by wdecoster** [comment]
You must decide if you want to use metrics from fastp, seqtk, or NanoPlot.
